src/components/button-toggle.tsx

```diff
@@ -0,0 +1,55 @@
+import { clsx } from "clsx";
+import React from "react";
+
+import { AppContext } from "../hooks/use-app-context";
+
+import "./button-toggle.scss";
+
+interface IButtonToggleProps {
+  ac: AppContext;
+  controlPrefix: string; // Used to generate keys for labels
+  invert?: boolean; // Set to true when the order of the boolean value should switch (true on the left)
+  leftClass?: string | boolean;
+  playVideo: () => void;
+  rightClass?: string | boolean;
+  setValue: (value: boolean) => void;
+  twoLines?: boolean; // Set to true to force the button's label to render as two lines
+  value: boolean;
+}
+export function ButtonToggle({
+  ac, controlPrefix, invert, leftClass, playVideo, rightClass, setValue, twoLines, value
+}: IButtonToggleProps) {
+  const leftSelected = invert ? value : !value;
+  const rightSelected = invert ? !value : value;
```

Comment:
The whole 'invert' thing really complicates this component. I'm not clear on why it's necessary,  though I suspect you had a good reason.